content/docs/code-splitting.md

```diff
@@ -147,29 +118,29 @@ function MyComponent() {
 }
 ```
 
-This will automatically load the bundle containing the `OtherComponent` when this component gets rendered.
+Cela chargera automatiquement le bundle contenant le composant `OtherComponent` quand celui-ci doit être rendu.
 
-`React.lazy` takes a function that must call a dynamic `import()`. This must return a `Promise` which resolves to a module with a `default` export containing a React component.
+`React.lazy` prend une fonction qui doit appeller un `import()` dynamique. Cela doit renvoyer une `Promise` résolue en un module avec un export par défault contenant un composant React.
 
 ### Suspense {#suspense}
 
-If the module containing the `OtherComponent` is not yet loaded by the time `MyComponent` renders, we must show some fallback content while we're waiting for it to load - such as a loading indicator. This is done using the `Suspense` component.
+Si le module contenant le composant `OtherComponent` n'est pas encore chargé au moment du rendu de `MyComponent`, alors nous devons afficher un contenu de repli en attendant qu'il soit chargé — comme un indicateur de chargement. Cela se fait en utilisant le composant `Suspense`.
```

Comment:
```suggestion
Si le module contenant le composant `OtherComponent` n'est pas encore chargé au moment du rendu de `MyComponent`, alors nous devons afficher un contenu de repli en attendant qu'il soit chargé—comme un indicateur de chargement. Ça se fait en utilisant le composant `Suspense`.
```

En pratique, la typographie française utiliserait un cadratin *entouré de deux fines insécables*, caractère qu'on ne peut aisément retranscrire sur le web. On pourrait les remplacer par des insécables, mais c'est 4x plus large, ça rend trop moche.  Dans tout le reste des trads validées, on a opté pour l'effet inverse : pas d'espaces autour du cadratin, qui en polisse à chasse proportionnelle est largement assez… large pour bien marquer l'incise. On a eu cette discussion au tel avec Jérémie lundi, pour ancrer la décision.

Donc je normalise.

Reply:
Ok c'est noté !

Reply:
OK mais tu n'as pas pris ma suggestion, on dirait…